deployments/scripts/setup-monitoring.sh

```
@@ -0,0 +1,366 @@
+#!/bin/bash
+set -euo pipefail
+
+# RBAC Token Service Monitoring Setup Script
+# This script sets up monitoring and alerting for the RBAC token service
+
+NAMESPACE="${NAMESPACE:-work-queue}"
+MONITORING_NAMESPACE="${MONITORING_NAMESPACE:-monitoring}"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "${SCRIPT_DIR}/lib/logging.sh"
```

Shellcheck (0.10.0)

[info] Not following: ./lib/logging.sh was not specified as input (see shellcheck -x).

(SC1091)